ingestion/src/metadata/data_quality/validations/table/pandas/tableRowInsertedCountToBeBetween.py

```diff
@@ -70,6 +73,19 @@ def _run_results(self, column_name: str, range_type: str, range_interval: int):
             range_type (str): range type (DAY, HOUR, MONTH, YEAR)
             range_interval (int): range interval
         """
-        threshold_date = self._get_threshold_date(range_type, range_interval)
+        import pandas as pd  # pylint: disable=import-outside-toplevel
 
-        return len(self.runner[0].query(f"{column_name} >= {threshold_date}"))
+        threshold_date = self._get_threshold_date(range_type, range_interval)
+        try:
+            return len(
+                pd.concat(
+                    runner.query(f"{column_name} >= {threshold_date}")
+                    for runner in self.runner
+                )
+            )
+        except MemoryError:
+            logger.error(
+                "Unable to compute due to memory constraints."
+                "We recommend using a smaller sample size or partitionning for the query."
+            )
+            return 0
```

Comment:
Same thing, don't need to aggregate it https://github.com/open-metadata/OpenMetadata/pull/10603#discussion_r1157219495

Reply:
This will work as expected, given that concat can come with memory error, added that exception as well..
to improve performance, let me sum the length..